### ISSUE TITLE
[native_pdf_renderer] Run project on Windows10

### ISSUE THREAD

**Issue by ggdream** [rbcprolabs/packages.flutter#235]
```
CMake Error at flutter/ephemeral/.plugin_symlinks/native_pdf_renderer/windows/DownloadProject.cmake:179 (message):
  Build step for pdfium failed: 1
Call Stack (most recent call first):
  flutter/ephemeral/.plugin_symlinks/native_pdf_renderer/windows/CMakeLists.txt:16 (download_project)
2

Exception: Unable to generate build files
```

Hello, How to  resolve it?

**Comment by SergeShkurko** [comment]
Fixed in #233 by @Jon-Salmon

**Comment by ggdream** [comment]
I only say it's awesome, thanks all.

So i close this.
